tests/test_install.py

```diff
@@ -375,3 +375,48 @@ def test_install_run_in_separate_directory(caplog, capsys, pipx_temp_env, monkey
     f.touch()
 
     install_packages(capsys, pipx_temp_env, caplog, ["pycowsay"], ["pycowsay"])
+
+
+@skip_if_windows
+@pytest.mark.parametrize(
+    "python_version",
+    [
+        str(sys.version_info.major),
+        f"{sys.version_info.major}.{sys.version_info.minor}",
+        f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}",
+    ],
+)
+def test_install_python_command_version(python_version):
+    assert not run_pipx_cli(["install", "--python", python_version, "--verbose", "pycowsay"])
+
```

Comment:
I'd check for the Python version used in the output here.

Reply:
Done.

I also suddenly realized the usage of `monkeypatch.setenv("PATH", ...)` here which causes `python3` not found in PATH when `pipx_temp_env` is used. The tests should be better now. Thanks for the reminder.